src/utils/utils_part.h

```diff
@@ -61,7 +62,26 @@ static inline void ocf_part_sort(struct ocf_cache *cache)
 	ocf_lst_sort(&cache->lst_part);
 }
 
-static inline ocf_cache_mode_t ocf_part_get_cache_mode(struct ocf_cache *cache,
+static inline bool ocf_part_occ_check_limit(struct ocf_request *req)
```

Comment:
This name could be confusing (for me it is), maybe rephrase it to something like `is_ocf_part_occ_reach_limit`.

Reply:
Yep, when I look at it I'm not sure if it should return `true` or `false` when the limit is reached. The name should indicate what the result means, not only what is checked.